benches/perf_ci/micro/dict_ops.py

```
@@ -0,0 +1,23 @@
+    d[i] = i
+    d["k%d" % (i % 512)] = i
+
+total = 0
+for i in range(N):
+    total += d[i]
+    total += d["k%d" % (i % 512)]
+
+for i in range(0, N, 2):
+    del d[i]
+
+for k in d:
```

Ruff (0.16.1)

[warning] Loop control variable `k` not used within loop body

Rename unused `k` to `_k`

(B007)